src/osd/osd_types.h

```diff
@@ -6606,4 +6606,56 @@ using missing_map_t = std::map<hobject_t,
   std::pair<std::optional<uint32_t>,
     std::optional<uint32_t>>>;
 
+/**
+ * Client QoS request parameters
+ *
+ */
+struct client_qos_params_t {
+  uint64_t reservation;
+  uint64_t weight;
+  uint64_t limit;
+
+  // A unique profile id for a given client. So a <client_id, qos_profile_id>
+  // tuple is a unique identifier for a client.
+  uint64_t qos_profile_id;
+
+  client_qos_params_t() :
+    reservation(1), weight(1), limit(999999), qos_profile_id(0)
```

Comment:
Set these defaults with the declaration.  limit should probably be std::numeric_limits<uint64_t>::max();

Reply:
```
  uint64_t reservation = 1;
  uint64_t weight = 1;
  uint64_t limit = std::numeric_limits<uin64_t>::max();
...
  client_qos_params_t() = default;
```

Reply:
Done.